uwsgi/django.wsgi

```diff
@@ -1,11 +1,16 @@
 # Generic WSGI application
 import os
+import uwsgi
 from django.core.wsgi import get_wsgi_application
 
 def application(environ, start):
-
     # copy any vars into os.environ
     for key in environ:
         os.environ[key] = str(environ[key])
 
-    return get_wsgi_application()(environ,start)
+    response = get_wsgi_application()(environ,start)
```

Comment:
Do we need to call `get_wsgi_application()` each time, or can it be done once. I imagine calling django's setup each time isn't necessary?